### DISCUSSION TITLE
how to login through tty?

### DISCUSSION THREAD

**Discussion by vyrx-dev** [prasanthrangan/hyprdots#1063] [Q&A] [answered]
how do I remove the lock screen and log in by using tty?

**Comment by kRHYME7** [comment, marked as answer]
``` systemctl disable sddm ```

**Reply by vyrx-dev** [reply]
if I disable sddm then  even after login into tty it shows the terminal only how to access WM and GUI

**Reply by kRHYME7** [reply]
ah, ``` Hyprland ```

**Reply by vyrx-dev** [reply]
do you know the solution?

**Reply by mahaveergurjar** [reply]
type Hyprland in terminal

**Reply by kRHYME7** [reply]
> do you know the solution?

Type ``` Hyprland ``` in the tty.
